docs/DATA_MODEL.md

```diff
@@ -171,6 +171,23 @@ Generated files created while building Tracker Radar
 |File|Use|
 |---|---|
 |[api_fingerprint_scores](/build-data/generated/api_fingerprint_scores.json)|An object mapping browser APIs to their likelihood to be used for fingerprinting. Higher score means that an API is more likely to be used for fingerprinting|
+|[tracking-parameters](/build-data/generated/tracking_parameters.json)|An object listing how prevalent a parameter is used in third party requests and cookies.|
+
+### Tracking parameters
+
+Tracking parameters are found by generating parameter values, and adding these to the crawl site URLs. When crawl data is processed these generated values can be found in cookies and third party requests.
+
+- **totalSites** - total number of sites processed
+- **params** -  an object listing request and cookie tracking for each parameter. Keys in the params object are the parameter name.
+  - prevalence - percent of totalSites that this parameter was seen in cookies or third party requests

```

Comment:
both first and third party cookies are taken into account here?

Reply:
Yes